src/styles/theme.scss

```diff
@@ -0,0 +1,20 @@
+//colors
+$hero-bkgd-color: #fed631;
+$green-section: #4f8d1f;
+$white: #fff;
+$redStripes: #cb463e;
```

Comment:
Make sure that your naming conventions are consistent. Choose between kebab case and camelCase.

Reply:
Sure, thank you